README.md

```diff
@@ -304,15 +304,22 @@ Be aware that running tests *will* power down running SecureDrop VMs, and may re
 
 ## Building the Templates
 
-1. Create a `fedora-29` AppVM for building
-2. Increase the disk size to at least 15GB (as the build uses over 10GB)
+1. Create a `fedora-29` AppVM for building the templates. It's going
+   to need Docker and several other packages every time you use it, so
+   it might be worth creating another template derived from
+   `fedora-29`, into which you can install those extras, and basing
+   the builder VM on that, or just using a StandaloneVM to save time
+   and repetition.
```

Comment:
Personally I don't bother to install docker inside the `sd-template-builder` VM; I simply `qvm-move <rpm>` the artifact back to my `sd-dev` environment for upload. Either way, your docs are certainly clearer than what we've had. Let's continue to discuss the optimal workflows here, and improve the docs as we go.